packages/frontend/src/pages/scaling-projects/view/ProjectHeader.tsx

```diff
@@ -60,5 +67,31 @@ export function ProjectHeader(props: ProjectHeaderProps) {
     },
   ]
 
+  if (props.ratingEnabled) {
+    const ratingStat = {
+      title: 'Stage',
+      value: props.ratingEntry ? (
+        <div className="flex items-center gap-2">
+          <RatingBadge
+            category={props.ratingEntry.category.score}
+            modifier={props.ratingEntry.modifier?.score}
+            small
+          />
+          <span
+            className="Tooltip"
+            title={renderToStaticMarkup(
+              <RatingTooltipPopup item={props.ratingEntry} />,
+            )}
+          >
+            <InfoIcon className="fill-gray-500 dark:fill-gray-600" />
+          </span>
+        </div>
+      ) : (
+        <NoDataCell />
+      ),
+    }
+    stats.splice(1, 0, ratingStat)
```

Comment:
This does look a little bit weird, instead of adding this object if ratingEnabled is true, we could conditionally add this stat to the array inside it
```
    },
    ...props.ratingEntry ? [{
      title: 'Stage',
      value: props.ratingEntry ? (
        <div className="flex items-center gap-2">
          <RatingBadge
            category={props.ratingEntry.category.score}
            modifier={props.ratingEntry.modifier?.score}
            small
          />
          <span
            className="Tooltip"
            title={renderToStaticMarkup(
              <RatingTooltipPopup item={props.ratingEntry} />,
            )}
          >
            <InfoIcon className="fill-gray-500 dark:fill-gray-600" />
          </span>
        </div>
      ) : (
        <NoDataCell />
      ),
    }] : [],
    {
``